ta/storage/Android.mk

```diff
@@ -1,6 +1,7 @@
+LOCAL_PATH := $(call my-dir)
+
 include $(CLEAR_VARS)
 LOCAL_MODULE := b689f2a7-8adf-477a-9f9932e90c0ad0a2.ta
```

Comment:
You're aware of that you can push push new changes to the same branch to update it? I.e., you don't have to close a pull request and create another one just to make changes.